src/scenes/AcceleratorRouter/NavBar.tsx

```diff
@@ -9,13 +10,18 @@ import LocaleSelector from 'src/components/LocaleSelector';
 import NavFooter from 'src/components/common/Navbar/NavFooter';
 
 type NavBarProps = {
-  setShowNavBar: (value: boolean) => void;
   backgroundTransparent?: boolean;
+  setShowNavBar: (value: boolean) => void;
 };
-export default function NavBar({ setShowNavBar, backgroundTransparent }: NavBarProps): JSX.Element | null {
+export default function NavBar({ backgroundTransparent, setShowNavBar }: NavBarProps): JSX.Element | null {
   const { isDesktop } = useDeviceInfo();
   const history = useHistory();
 
+  const isAcceleratorDeliverablesRoute = useRouteMatch(APP_PATHS.ACCELERATOR_DELIVERABLES + '/');
```

Comment:
Are these suffixed `/` needed? Seems like we should be getting sent to the route without a trailing slash. I vaguely remember some redirect code in the regular side of terraware that does some slash to no slash (or the other way around) redirecting.
 
Since we aren't using `exact` in the `Route` components, we should be able to have trailing slash and no slash go to the same place.

Reply:
Good catch! I copied the pattern from `TerrawareRouter/NavBar`, but the trailing slash isn't needed here. Will remove.